packages/central-server/src/apiV2/middleware/logApiRequest.js

```diff
@@ -8,6 +8,8 @@ export const logApiRequest = async (req, res, next) => {
   const refreshToken = await extractRefreshTokenFromReq(req);
   const { id: apiRequestLogId } = await req.models.apiRequestLog.create({
     version: 2, // only version that can get through current route handling
+    api: 'central',
+    method: req.method,
```

Comment:
Tracking the request method is pretty useful, since there are a lot of routes that use the same endpoints but have different behaviour for GET/POST etc.